libp2p/Host.h

```diff
@@ -135,13 +135,7 @@ class Host: public Worker
     static std::unordered_map<Public, std::string> pocHosts();
 
     /// Register a host capability; all new peer connections will see this capability.
-    void registerCapability(std::shared_ptr<CapabilityFace> const& _cap);
-
-    /// Register a host capability with arbitrary name and version.
-    /// Might be useful when you want to handle several subprotocol versions with a single
```

Comment:
I even wrote this explaining comment :cry:

Reply:
Restored.